client/js/verification-portal/pregnancy-resource-center/form.js

```diff
@@ -188,52 +193,58 @@ class VerificationPortalForm extends Component {
 							/>
 						</div>
 					</div>
-				</div>
 
-				{/* 5-12-18, I am commenting this out for now, as we want to keep the
+					{/* </div> */}
+
+					{/* 5-12-18, I am commenting this out for now, as we want to keep the
 				data verification simple for now. We will eventually add this back. */}
-				{/*<div>*/}
-				{/*<h3>Primary Contact</h3>*/}
-				{/*<Field*/}
-				{/*label='First Name'*/}
-				{/*name='primaryContactPerson.firstName'*/}
-				{/*component={Input}*/}
-				{/*type='text'*/}
-				{/*/>*/}
-
-				{/*<Field*/}
-				{/*label='Last Name'*/}
-				{/*name='primaryContactPerson.lastName'*/}
-				{/*component={Input}*/}
-				{/*type='text'*/}
-				{/*/>*/}
-
-				{/*<Field*/}
-				{/*label='Email'*/}
-				{/*name='primaryContactPerson.email'*/}
-				{/*component={Input}*/}
-				{/*type='text'*/}
-				{/*/>*/}
-
-				{/*<div className={classes.parent}>*/}
-				{/*<Field*/}
-				{/*label='Phone Number'*/}
-				{/*name='primaryContactPerson.phone'*/}
-				{/*placeholder='Format must be +19998887777'*/}
-				{/*component={Input}*/}
-				{/*type='text'*/}
-				{/*/>*/}
-				{/*<Field*/}
-				{/*label='Primary Contact Verified'*/}
-				{/*name='verifiedData.primaryContactPerson.verified'*/}
-				{/*component={VerifiedCheckbox}*/}
-				{/*/>*/}
-				{/*</div>*/}
-				{/*</div>*/}
-
-				<Spacer height="83px" />
-				<div className={classes.formSection}>
-					<Heading text="SERVICES" />
+					{/*<div>*/}
+					{/*<h3>Primary Contact</h3>*/}
+					{/*<Field*/}
+					{/*label='First Name'*/}
+					{/*name='primaryContactPerson.firstName'*/}
+					{/*component={Input}*/}
+					{/*type='text'*/}
+					{/*/>*/}
+
+					{/*<Field*/}
+					{/*label='Last Name'*/}
+					{/*name='primaryContactPerson.lastName'*/}
+					{/*component={Input}*/}
+					{/*type='text'*/}
+					{/*/>*/}
+
+					{/*<Field*/}
+					{/*label='Email'*/}
+					{/*name='primaryContactPerson.email'*/}
+					{/*component={Input}*/}
+					{/*type='text'*/}
+					{/*/>*/}
+
+					{/*<div className={classes.parent}>*/}
+					{/*<Field*/}
+					{/*label='Phone Number'*/}
+					{/*name='primaryContactPerson.phone'*/}
+					{/*placeholder='Format must be +19998887777'*/}
+					{/*component={Input}*/}
+					{/*type='text'*/}
+					{/*/>*/}
+					{/*<Field*/}
+					{/*label='Primary Contact Verified'*/}
+					{/*name='verifiedData.primaryContactPerson.verified'*/}
+					{/*component={VerifiedCheckbox}*/}
+					{/*/>*/}
+					{/*</div>*/}
+					{/*</div>*/}
+
+					<Spacer height="50px" />
```

Comment:
This space should be 81px

<img width="344" alt="screen shot 2018-07-03 at 2 43 41 pm" src="https://user-images.githubusercontent.com/6378435/42241276-972a8170-7ecf-11e8-9ceb-ac47199d350d.png">